modules/ROOT/pages/advanced-mule-concepts.adoc

```diff
@@ -0,0 +1,17 @@
+= Advanced Mule Concepts
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: advanced, mule, concepts, runtime, engine
+
+This section covers some advanced Mule topics that will be useful once you are familiar with the basics of Mule application development and Mule runtime engine concepts.
+
+Learn about the xref:execution-engine.adoc[Execution Engine] to understand how the Mule runtime engine processes the operations in a Mule application’s flow, and understand how to change the thread pools configuration to optimize performance.
```

Comment:
The phrase `change the thread pools configuration to optimize performance` is contradictory with the self-tuning approach that is already part of our message. I would rephrase it to just `change the thread pools configuration if needed`